docs/source/getting-started/model-registry/step3-model-version-alias.rst

```diff
@@ -0,0 +1,77 @@
+Alias Model Version
+===================
+
+Model version aliases provide a flexible way to set named aliases on model versions. For example, 
+setting a champion alias on a model version enables you to fetch this model version by that alias 
+via client API :py:func:`mlflow.client.MlflowClient.get_model_version_by_alias()` or the model URI 
+`models:/<registered model name>@champion`. Aliases can be easily reassigned to new model versions 
+via the UI and client API alike, thereby decoupling model deployment from the production system 
+code. Unlike model registry stages, more than one alias can be applied to any given model version, 
+creating powerful possibilities for model deployment.
+
```

Comment:
Let's also provide links to the new section that @jerrylian-db has created for aliases in the registry docs so that users can go and read a bit more if they're inclined.